### ISSUE TITLE
Can we add a "coreutils listutils" entry to help with automated installations?

### ISSUE THREAD

**Issue by alshdavid** [uutils/coreutils#6466]
I like to install uutils by hand from the binary. To do this I wget the latest binary and create symlinks for each util within coreutils

The output of `coreutils` at the moment is:

```
$ ./coreutils
coreutils 0.0.26 (multi-call binary)

Usage: coreutils [function [arguments...]]

Currently defined functions:

    [, arch, b2sum, b3sum, base32, base64, basename, basenc, cat, chgrp, chmod, chown, chroot,
    cksum, comm, cp, csplit, cut, date, dd, df, dir, dircolors, dirname, du, echo, env, expand,
    expr, factor, false, fmt, fold, groups, hashsum, head, hostid, hostname, id, install, join,
    kill, link, ln, logname, ls, md5sum, mkdir, mkfifo, mknod, mktemp, more, mv, nice, nl,
    nohup, nproc, numfmt, od, paste, pathchk, pinky, pr, printenv, printf, ptx, pwd, readlink,
    realpath, rm, rmdir, seq, sha1sum, sha224sum, sha256sum, sha3-224sum, sha3-256sum, sha3-
    384sum, sha3-512sum, sha384sum, sha3sum, sha512sum, shake128sum, shake256sum, shred, shuf,
    sleep, sort, split, stat, stdbuf, stty, sum, sync, tac, tail, tee, test, timeout, touch,
    tr, true, truncate, tsort, tty, uname, unexpand, uniq, unlink, uptime, users, vdir, wc, who,
    whoami, yes
```
Which is a little hard to parse. Could we add a `coreutils listutils` function that prints the utils with a newline separator?
```
$ ./coreutils listutils
arch
b2sum
b3sum
base32
base64
basename
...
```
This would allow me to write a simplified script that automates the generation of symlinks from a bash/powershell script

**Comment by cakebaker** [comment]
It's already there: `coreutils --list`

**Comment by alshdavid** [comment]
Am I missing something? 
![image](https://github.com/uutils/coreutils/assets/12656294/adcf2ac2-f1cd-4abc-ad0e-8013dce72558)

**Comment by tertsdiepraam** [comment]
https://github.com/uutils/coreutils/pull/6251

It's fairly new so you might not have a version with that feature yet.

**Comment by JoshMcguigan** [comment]
This is great! 

Does uutils/coreutils have a policy on tagging new versions? A new tagged release would help pull this into package managers. 

Either way, thanks for implementing this!

**Comment by sylvestre** [comment]
> Does uutils/coreutils have a policy on tagging new versions?

when someone asks :)
and around every two months

**Comment by JoshMcguigan** [comment]
![image](https://github.com/uutils/coreutils/assets/22216761/cc6d108f-7d26-45d4-83aa-6bf604c8f9ba)

Thanks for tagging `0.0.27`!

**Comment by sylvestre** [comment]
welcome :)
